reduction/imaging_parameters.py

```diff
@@ -2999,17 +2999,23 @@
     else:
         line_imaging_parameters[key] = line_imaging_parameters_custom[key]
 
+
 default_lines = {
+    "h41a": "92.034434GHz",
+    "ch3cn": "92.26144GHz",
```

Comment:
the CH3CN line is, iirc, the middle-ish of the K-ladder

Reply:
![CH3CN_B3_splatalogue](https://user-images.githubusercontent.com/12817147/102623276-dcce0d80-4107-11eb-9c69-c1a77a1be62c.png)

Reply:
I don't think so.

Reply:
you're right, it's ch3cn v8=1, which perhaps isn't such a likely detection